### ISSUE TITLE
Create link on other field than string

### ISSUE THREAD

**Issue by AlysG** [dmitrios/grafana-plugins-reftable#3]
Thanks a lot for this plugin!
In my case, on renderer.js (l63), the result of "lodash_1.default.isString(v)" is false so I have no link. I commented the line and it works. This test is necessary?

**Comment by Kenith** [comment]
Hello AlysG,

I have same issue as you, and this column disappears if adding link type.
How do you fix that?
I tried to search the lodash_1.default.isString(v), but nothing found, would you please explain more?

really appreciate for your help.

Thank you,
Ken

**Comment by Kenith** [comment]
Hello AlysG,

I have fixed that. Wrong put from my side.
@dmitrios : Really appreciate for your great work!!

Thank you,
Ken

**Comment by dmitrios** [comment]
Yes, checking for the string should not be necessary, any non-string value can used to generate a link. Please feel free to submit PR if you'd like.
